src/themes/default/components/core/blocks/Checkout/OrderReview.vue

```diff
@@ -63,8 +63,10 @@
       <div class="col-xs-12 col-sm-9 col-md-11">
         <div class="row">
           <div class="col-xs-12 col-md-8 px20">
+            <div v-show="!placeOrderVisible" id="place-order-container"/>
```

Comment:
This div could be visible all the time but empty / cleared each time when user starts the checkout. Then if some method will add content to i (for example the PayPal button), default button should be hidden automatically